extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -467,6 +468,31 @@ tags: {
   }
   spec_url: "https://www.ampproject.org/docs/reference/components/amp-story"
 }
+tags: { #<amp-sidebar> not in amp-story
```

Comment:
I think this will need to go in [validator-amp-sidebar.protoascii](https://github.com/ampproject/amphtml/blob/master/extensions/amp-sidebar/validator-amp-sidebar.protoascii)

Reply:
I moved all of the code to validator-amp-sidebar.protoascii, since that's what we did with [amp-video](https://github.com/ampproject/amphtml/blob/96e86fd50e6aed8bb36b9799ebd3581984c1d4e7/extensions/amp-video/validator-amp-video.protoascii#L103-L155)